### ISSUE TITLE
Missing TS declarations

### ISSUE THREAD

**Issue by rozsival** [jiripudil/Naja#198]
# Bug Report

It seems that since `v2.1.0` all `.d.ts` files are missing from the repository.

## Types

https://github.com/jiripudil/Naja/blob/e33269152ea070c9207cfc512c4b8a37c41e602a/package.json#L16

## TSC output

```bash
error TS7016: Could not find a declaration file for module 'naja'. '~/node_modules/naja/dist/Naja.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/naja` if it exists or add a new declaration (.d.ts) file containing `declare module 'naja';`
```

## Compare
https://github.com/jiripudil/Naja/compare/2.0.1...2.1.0

**Comment by jiripudil** [comment]
Oh yes, nice catch, thanks! That should be `dist/`, not `src/`

**Comment by jiripudil** [comment]
Fixed in [2.1.2](https://github.com/jiripudil/Naja/releases/tag/2.1.2)
